pkg/inferenceservice-controller/controller.go

```diff
@@ -242,7 +244,7 @@ func (r *InferenceServiceController) getMRUrlFromService(ctx context.Context, na
 		return "", fmt.Errorf("unable to find the http port in the Model Registry Service")
 	}
 
-	return fmt.Sprintf("%s.%s.svc.cluster.local:%d", name, namespace, *restApiPort), nil
+	return fmt.Sprintf("http://%s.%s.svc.cluster.local:%d", name, namespace, *restApiPort), nil
```

Comment:
wouldn't hardcoding `http://` be a problem midstream/downstream of Kubeflow where Distributions may default to mTLS? 🤔

Reply:
I would leave it like that for a default configuration and let the user decide using the `modelregistry.kubeflow.org/url` for every other scenario mLTS/external route/etc